suites/pacific/iscsi/tier_0_iscsi.yaml

```diff
@@ -0,0 +1,121 @@
+tests:
```

Comment:
Do we need separate tier-0 for iscsi. 

Currently we only qualify build with Tier-0. 
IMO, It would be good if we add this as part of tier-1 or above.

Reply:
After discussing made changes as Tier-1 for ISCSI functionality test cases.